src/antenna.ts

```diff
@@ -4,13 +4,19 @@ import { IRpcMethod } from "./rpc-method/types";
 
 export type Opts = {
   signer?: SignerPlugin;
+  timeout?: number;
+  token?: string;
```

Comment:
how do you think `apiToken` because there are jwt and iotx, etc20 tokens here in this package. too many tokens and we should differentiate them